packages/suite/src/actions/firmware/__fixtures__/firmwareActions.ts

```diff
@@ -92,17 +92,17 @@ export const actions = [
         },
         result: {
             actions: [
-                { type: FIRMWARE.SET_UPDATE_STATUS, payload: 'started' },
-                { type: FIRMWARE.SET_INTERMEDIARY_INSTALLED, payload: true },
-                { type: FIRMWARE.SET_HASH, payload: firmwareUpdateResponsePayload },
-                { type: FIRMWARE.SET_UPDATE_STATUS, payload: 'unplug' },
+                { type: firmwareActions.setStatus.type, payload: 'started' },
+                { type: firmwareActions.setIntermediaryInstalled.type, payload: true },
+                { type: firmwareActions.setHash.type, payload: firmwareUpdateResponsePayload },
+                { type: firmwareActions.setStatus.type, payload: 'unplug' },
             ],
             state: { firmware: { status: 'unplug', error: undefined } },
         },
     },
     {
         description: 'Success T1 (without intermediary)',
-        action: () => firmwareActions.firmwareUpdate(),
+        action: () => firmwareUpdate(),
```

Comment:
why not `firmwareActions.firmwareUpdate()`?

Reply:
I was also thinking about this. This should actually be a thunk and named properly so it doesn't need to be exported in the `firmwareActions` namespace IMO. So I would sugest to rename it as `firmwareUpdateThunk` to follow convention from other packages but drop the namespace as it is now in the diff. What do you think?

Reply:
So `firmwareActions` should only contain plain actions and not thunks? Because it used to not matter. And since they are still in the same file, they might as well be imported from the same namespace :D

Reply:
Yes, but now we use the wrapper object to infer types of actions instead of predefined actions types that were found in this file. 

`type FirmwareAction = ReturnType<(typeof firmwareActions)[keyof typeof firmwareActions]>;`

This is an example of that and adding thunks to it would break types of redux related logic which is unwanted. In my opinion, there is no reason to have it in the same namespace. What we could do, if you want, is to create a file `firmwareThunks.ts` and move these thunk actions there with proper naming so they don't need to be part of this file and it will make it a bit more clean in the end. Same solution is used in `suite-common/wallet-core` packages so it's actually a good idea to do this but I'd prefer to do that in a separate PR not to bloat this diff if you agree with that.

Reply:
I see that you have files like `accountsThunks.ts` already, so yeah, even though it's just 1 func, since there is some time magic tied to `firmwareActions.ts`, I'd not mix those 🙏

Reply:
https://github.com/trezor/trezor-suite/pull/8864/commits/b970d4f77dff1657ad20ea26e6ea1e58dc6bc9d5